Sources/Plasma/PubUtilLib/plModifier/plImageLibMod.cpp

```diff
@@ -97,3 +97,23 @@ void plImageLibMod::Write(hsStream* stream, hsResMgr* mgr)
     for (plBitmap* image : fImages)
         mgr->WriteKey(stream, image->GetKey());
 }
+
+plBitmap* plImageLibMod::GetImage(const ST::string& imageName) const
+{
+    for(auto image = fImages.begin(); image < fImages.end(); image++) {
+        if ((*image) && (*image)->GetKeyName() == imageName)
+            return *image;
+    }
```

Comment:
```suggestion
```suggestion
    auto findIt = std::find_if(fImages.begin(), fImages.end(), [&imageName](plBipmap* x) { x->GetKeyName() == imageName });
    if (findIt != fImages.end())
        return *findIt;
```
```